Models/userModel.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('../DataBase/connection');
+const UserSchema = mongoose.Schema({
```

Comment:
Need to either instantiate a new instance of mongoose.Schema, or declare a global Schema variable.
- instantiate:  const UserSchema = new mongoose.Schema({
- declare:  const Schema = mongoose.Schema;
                   const UserSchema = new Schema({